flink-table/flink-table-planner/src/test/scala/org/apache/flink/table/planner/plan/stream/sql/join/WindowJoinTest.scala

```diff
@@ -388,6 +388,7 @@ class WindowJoinTest extends TableTestBase {
         |    a,
         |    window_start,
         |    window_end,
+        |    window_time,
```

Comment:
Those unit tests in `WindowJoinTest` would fallback to the regular join because they don't satisfy the condition to convert to window join. In the previous version, if the input contains time attributes, exception would be thrown out. So in the previous pr, I skip the time attributes in the select clause. 
I add back the time attributes in the select clause to test now join input could contain time attributes fields because they would materialized in `RelTimeIndicatorConveter ` now.